WcaOnRails/app/views/admin/_fix_results_selector.html.erb

```diff
@@ -0,0 +1,108 @@
+<div id="fix-results-selector">
+  <%= simple_form_for @result_selector, url: @result_selector.result_or_repeat_link(self), method: :get, html: { id: "fix-selector-form" } do |f| %>
+    <%= f.input :person_id, as: :string, label: "WCA ID", hint: @result_selector.person_hint(self) %>
+
+    <div id="selector-result-details">
+      <% if @result_selector.person
+           # NOT querying person_id here, to make sure that user input exists %>
+        <% if @result_selector.eligible_competitions.any? %>
+          <%= f.input :competition_id, collection: @result_selector.eligible_competitions, value_method: :id, label_method: :name, label: "Competition", hint: false %>
```

Comment:
Oh right now seeing the code my suggestion on slack to be able to "fuzzy search" through competitions would definitely be too much work!
I thought this was using react and semantic ui and it would have been easier ^^